sway-core/Cargo.toml

```
@@ -18,6 +18,7 @@ dirs = "3.0"
 either = "1.6"
 fuel-asm = "0.2"
 fuel-vm = "0.5"
+fuel-tx = "0.5.0"
```

Also alphabetize deps, this should be on top of `fuel-vm`
```suggestion
fuel-tx = "0.6"
```

---

I guess I need `fuel-crypto` instead of `fuel-tx` now. I need the `hasher` basically.